staging/src/k8s.io/apiserver/plugin/pkg/audit/dynamic/dynamic.go

```diff
@@ -336,7 +336,7 @@ func (b *backend) createAndStartDelegate(sink *auditregv1alpha1.AuditSink) (*del
 func (b *backend) String() string {
 	var delegateStrings []string
 	for _, delegate := range b.GetDelegates() {
-		delegateStrings = append(delegateStrings, fmt.Sprintf("%s", delegate))
+		delegateStrings = append(delegateStrings, fmt.Sprintf("%s", delegate.(string)))
```

Comment:
if it is a string, we don't need Sprintf. Is it always a string? Isn't this just calling the `String` method?

Reply:
The type is interface.
It should do type assertion when using Sprintf.

Reply:
if delegate is not string (say nil), the type assertion will panic

Reply:
I think the best way is using %v instead of %s, because delegate type is an interface.

Reply:
Or rethink what a delegate actually is. Make it a non-anonymous interface.